.travis.yml

```diff
@@ -1,11 +1,16 @@
-dist: trusty
 sudo: required
 language: node_js
 node_js:
-- '8'
-- '10'
+  - '8'
+  - '10'
```

Comment:
add 12?

Reply:
will do, right now travis is flaky as hell tho